### PR TITLE
Contrib: add an example for running rethinkdb cluster on k8s

### PR DESCRIPTION
**Pull request by antmanler**
Added an example to setup a [rethinkdb](http://rethinkdb.com/) cluster.

> RethinkDB is built to store JSON documents, and scale to multiple machines with very little effort.

I added a bash script which will query from kubernetes_ro service to find other peers and start server.

### CODE REVIEW

File: examples/rethinkdb/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+```shell
+$kubectl get se
+NAME                LABELS              SELECTOR                  IP                  PORT
+rethinkdb-admin     db=influxdb         db=rethinkdb,role=admin   10.241.220.209      8080
+rethinkdb-driver    db=influxdb         db=rethinkdb              10.241.105.47       28015
+```
+
+open a web browser and access to *http://10.241.220.209:8080* to manage you cluster
+
+**Why not just using pods in replicas?**
```
Comment (brendandburns) [inline examples/rethinkdb/README.md]: I think that if you set session affinity in the service, you might not have this problem.

**Comment by brendandburns** [comment]
LGTM.  Thanks for the contribution!  Unfortunately, our lawyers require that you sign our CLA before we can merge.  You can find details in [CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/CONTRIBUTING.md)

Thanks again!
--brendan

File: examples/rethinkdb/README.md (hunk 1 above)
Comment (antmanler) [inline examples/rethinkdb/README.md]: Got it, I'll try it, thanks

**Comment by antmanler** [comment]
I've signed CLA :)

**Comment by brendandburns** [comment]
Thanks.  Merged.

**Pull request merged**